### PR TITLE
chore: synthetic file generation for tests

### PR DESCRIPTION
**Pull request by rostislavdeepcrawl**

### CODE REVIEW

File: src/tests/perf-synthetic.test.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console, jest/expect-expect */
+import { join } from "path";
+
+import { Connection, DuckDB } from "@addon";
+
+import { writeSyntheticParquetFile } from "./synthetic-test-data-generator";
+
+const filePath = join(__dirname, "../large-synth");
```
Comment (jupiter) [inline src/tests/perf-synthetic.test.ts:8]: Should we exclude this location from .gitignore?

File: src/tests/synthetic-test-data-generator/index.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console, no-loops/no-loops, no-restricted-properties, max-lines-per-function */
+import { access, constants, unlink } from "fs";
+import { promisify } from "util";
+
+import { random, times } from "lodash";
+import parquet from "parquetjs";
+
+const accessAsync = promisify(access);
```
Comment (jupiter) [inline src/tests/synthetic-test-data-generator/index.ts:8]: These are already available as Promise-based functions in `fs.promises`.

**Review by jupiter — CHANGES_REQUESTED** [review]

File: src/tests/perf-synthetic.test.ts (hunk 1 above)
Comment (rostislavdeepcrawl) [reply src/tests/perf-synthetic.test.ts:8]: it's already inside `dist` which is excluded

Comment (jupiter) [reply src/tests/perf-synthetic.test.ts:8]: Ah, okay

**Review by jupiter — APPROVED** [review]

**Pull request merged**